.github/workflows/inductor-rocm.yml

```diff
@@ -0,0 +1,49 @@
+name: inductor-rocm
+
+on:
+  schedule:
+    # We have several schedules so jobs can check github.event.schedule to activate only for a fraction of the runs.
+    # Also run less frequently on weekends.
+    - cron: 45 0,8,16 * * 1-5
```

Comment:
Do we need so many schedule here?  They come from the current periodic workflow, in this case I think we just want to run every 4 hours and keep the daily mem leak check and rerun disabled tests one